lib/rubocop/cop/rspec/subject_stub.rb

```diff
@@ -94,7 +94,7 @@ def processed_example_groups
         end
 
         def find_all_explicit_subjects(node)
-          node.each_descendant(:block).each_with_object({}) do |child, h|
+          node.each_descendant(:block).with_object({}) do |child, h|
```

Comment:
Today I learned about `with_object`. Hadn’t noticed that alias before. And it’s been there since 2.4‽

Reply:
More like 1.9.2 :D https://ruby-doc.org/core-1.9.2/Enumerator.html#method-i-with_object